h/views/activity.py

```diff
@@ -130,6 +134,11 @@ def user_annotation_count(aggregation, userid):
 
     result['more_info'] = 'more_info' in request.params
 
+    if not result.get('q'):
+        result['zero_message'] = Markup(
+            _('The group &ldquo;{name}&rdquo; have not made any annotations '
```

Comment:
I've put the unicode characters in. I've done the message tweak you suggested as well, which differs from in Conor's mockup. @conordelahunty if _has not_ is US and _have not_ is British, this seems a clear rule to follow as Rob suggests?